docs/src/pages/[platform]/connected-components/authenticator/advanced/example.react-native.mdx

```diff
@@ -3,9 +3,6 @@ import { View, Button, Text } from 'react-native';
 import { Amplify } from 'aws-amplify';
 import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
 
-import awsExports from './src/aws-exports';
```

Comment:
My thinking here is that these are extraneous to the example and should already be present